ui_framework/components/toggle_button/_toggle_panel.scss

```diff
@@ -0,0 +1,5 @@
+.kuiTogglePanelHeader {
+  padding-bottom: 4px;
+  margin-bottom: 15px;
+  border-bottom: $globalBorderThin;
+}
```

Comment:
What if we added a rule here that made the button full-width, so that you can just click anywhere in the header to open and close it? I think this is a slightly better UX.

```scss
.kuiTogglePanelHeader {
  padding-bottom: 4px;
  margin-bottom: 15px;
  border-bottom: $globalBorderThin;

  /**
   * 1. Allow the user to click anywhere on the header, not just on the button text.
   */
  .kuiToggleButton {
    width: 100%; /* 1 */
    text-align: left; /* 1 */
  }
}
```